### DISCUSSION TITLE
Compilation and execution of Spinodal Decomposition Tutorial

### DISCUSSION THREAD

**Discussion by rsiddu0511** [idaholab/moose#19933] [Q&A Modules: Phase field] [answered]
Hello dear members,
    I am a novice to using MOOSE and am interested to model Spinodal decomposition. While I am entirely convinced about the implementation, thanks to the elaborate details provided, I am not sure how to execute the final 's5_energycurve.i' as there are not make files associated with it in the directory. Any kind of help/hints would be really helpful. Thanking in advance.

**Comment by ykvishal** [comment, marked as answer]
You can try this suggestion: Go to `moose/modules/phase_field/` folder and run  `make -j8`. Now, try this opt file to run input files from 
 `moose/modules/phase_field/tutorials/spinodal_decomposition/`

In my view, it is better to create [an app](https://mooseframework.inl.gov/getting_started/new_users.html#create-an-app) and run all your simulations.

**Reply by rsiddu0511** [reply]
Thank you very much.
